src/components/input/index.jsx

```diff
@@ -112,6 +112,9 @@ class Input extends React.Component {
             label,
             name,
             maxLength,
+            numMax,
+            numMin,
+            numStep,
```

Comment:
How do you feel about naming these `max`, `min`, `step` to make the interface similar to using a plain input in HTML